src/components/templates/PartyMap/components/ScheduleItem/ScheduleItem.tsx

```diff
@@ -29,28 +29,25 @@ export const ScheduleItem: React.FunctionComponent<PropsType> = ({
   return (
     <div className="schedule-item-container">
       <div className={`time-section ${isCurrentEvent ? "primary" : ""}`}>
-        <div>
-          <b>{formatUtcSeconds(event.start_utc_seconds)}</b>
-        </div>
-        <div>
+        <span className="event-date">
+          {labelDayRoomSchedule(event.start_utc_seconds)}
+        </span>
+        <span className="event-time">
+          {formatUtcSeconds(event.start_utc_seconds)}
+        </span>
+        <span className="event-time">
           {formatUtcSeconds(
             event.start_utc_seconds + event.duration_minutes * 60
           )}
-        </div>
+        </span>
       </div>
       <div className="event-section">
-        <div>
-          <div className={`${isCurrentEvent ? "primary" : ""}`}>
-            <div>
-              <b>{event.name}</b>
-            </div>
-            <div>
-              by <b>{event.host}</b>
-            </div>
-          </div>
-          <div className="event-description">
+        <div className={`${isCurrentEvent ? "primary" : ""}`}>
+          <span className="event-name">{event.name}</span>
+          by <span className="event-host">{event.host}</span>
```

Comment:
I guess we should do the same check here as we do in the `ScheduleEvent` component? 
(Note: the class name should be updated too, see my another comment)

```suggestion
          {event.host && (
            <>
              by <span className="event-host">{event.host}</span>
            </>
          )}
```

Reply:
I'm fine with putting that in, but @0xdevalias suggested I remove that, as `event.host` is a required field. Just let me know whether to implement, and I'll do it

Reply:
Leave it without IMO. I also called out another location to remove it from in my most recent review as well.